cpg-language-python/src/test/resources/python/with.py

```diff
@@ -0,0 +1,16 @@
+with open("file.txt", "r") as file:
```

Comment:
Can we please also have a test with multiple resources? Something along the line
```python
with A() as a, B(a) as b, C(a, b) as c:
    doSomething(a, c)
```
See https://stackoverflow.com/a/1073814

Reply:
Hm, this requires nesting the `with` statements. Currently, this is not done correctly

Reply:
I think this is already implemented as you are looping over the "node.items". The above example has 3 "items".

Reply:
Done